File: rest/src/main/java/com/droidtitan/volleyexamples/rest/model/air/AirQuality.java
package com.droidtitan.volleyexamples.rest.model.air;

import com.google.gson.annotations.SerializedName;

public class AirQuality {

    @SerializedName("categoria")
    String category;

    @SerializedName("color")
    String color;

    @SerializedName("recomendaciones")
    String recommendation;

    public String getCategory() {
        return category;
    }

    public String getColor() {
        return color;
    }

    public String getRecommendation() {
        return recommendation;
    }
}

File: app/src/main/java/com/droidtitan/volley/model/air/AirQuality.kt
package com.droidtitan.volley.model.air

import com.google.gson.annotations.SerializedName

class AirQuality {

  @SerializedName("categoria") var category: String? = null

  @SerializedName("color") var color: String? = null

  @SerializedName("recomendaciones") var recommendation: String? = null
}
